### PR TITLE
[wip]: bug 1798586: pkg/*: Remove support for Unmanaged

### PR DESCRIPTION
**Pull request by retroflexer**
Remove any obedience of being `Unmanaged`. We are always `Managed`.

### CODE REVIEW

**Comment by sttts** [comment]
Add a restriction to the config API. Compare how kube-apiserver does that via https://github.com/openshift/api/blob/master/operator/v1/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch.

File: manifests/0000_12_etcd-operator_01_config.crd.yaml [hunk 1]
```diff
@@ -14,3 +14,10 @@ spec:
     - coreoperators
   subresources:
     status: {}
+  validation:
```
Comment (retroflexer) [inline manifests/0000_12_etcd-operator_01_config.crd.yaml]: Based on @sttts' suggestion here:
https://github.com/openshift/cluster-etcd-operator/pull/83#issuecomment-582351754

Comment (sttts) [reply manifests/0000_12_etcd-operator_01_config.crd.yaml]: From our slack discussion:

The CRD should not be in the ceo operator, but in openshift/api. Then you also get the schema generation for free (there is a makefile target for that). And you have to follow the approach kube-apiserver, i.e. using a yaml patch.

Comment (sttts) [reply manifests/0000_12_etcd-operator_01_config.crd.yaml]: cc @damemi – maybe you can help @retroflexer with moving the CRD manifest over.

Comment (retroflexer) [reply manifests/0000_12_etcd-operator_01_config.crd.yaml]: I will move the CRD manifest to openshift/api in a separate PR. I would let this PR go through without the CRD changes (I reverted the above change).

**Comment by retroflexer** [comment]
Opened a new PR to use the latest rebase. Closing this.
https://github.com/openshift/cluster-etcd-operator/pull/89

**Pull request closed without merging**